packages/extension-app/package.json

```diff
@@ -12,8 +12,8 @@
     "start": "webpack --watch"
   },
   "dependencies": {
-    "@polkadot/extension-base": "^0.24.0-beta.7",
-    "@polkadot/extension-chains": "^0.22.0-beta.10",
+    "@polkadot/extension-base": "^0.23.1",
```

Comment:
I'm using some stuff only in beta, would rather keep 0.24-beta

Reply:
Ill keep it at the previous beta (.7, .10), however just a heads up it looks like Jaco just removed the `ExtensionStore` in the latest beta (.15)